bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/DahuaHandler.java

```diff
@@ -352,6 +437,16 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                             .sendHttpGET("/cgi-bin/configManager.cgi?action=setConfig&MotionDetect[0].Enable=false");
                 }
                 return;
+            case CHANNEL_MOTION_DETECTION_LEVEL:
+                int motionlevel = Math.round(Float.valueOf(command.toString()));
```

Comment:
This channel is a DIMMER so this line of code will fail if ON or OFF commands are sent which are valid for this channel type. You could use something like the following `if (command instanceof DecimalType)`, but a better way would be not to turn into a String and then parse and round it if a less costly approach can be used.

Reply:
Tests required to set actually usable Motion Detection Level limits. I've found out for my VTO that I do need levels from 1 to 4, and level 4 is extremely sensitive in my case. But my VTO is placed outdoors, and objects max distance from camera is something like 10 meters. Could you test those settings with your camera? If you'll find out the same results, item's type could be changed to type Number, limited from 1 to 4.

Reply:
Sorry I do not own a VTO camera, is that a requirement to test?